lib/bolt_ext/server.rb

```diff
@@ -51,8 +51,10 @@ def initialize(app = nil)
     schema_error = JSON::Validator.fully_validate(@schemas["ssh-run_task"], body)
     return [400, schema_error.join] if schema_error.any?
 
-    keys = %w[user password port ssh-key-content connect-timeout run-as-command run-as
-              tmpdir host-key-check known-hosts-content private-key-content sudo-password]
+    # CODEREVIEW: the schema is additionalProperties false do we need this?
```

Comment:
Probably not then.

Reply:
We will delete this next PR. Going to merge now to get change through for Console team.